gapminders/anaconda-project.yml

```
@@ -10,24 +10,23 @@ examples_config:
   - panel
   deployments:
   - command: dashboard
-  no_data_ingestion: true
 
 user_fields: [examples_config]
 
 channels:
 - defaults
 
 packages: &pkgs
-- python=3.9
-- notebook
-- plotly
-- holoviews
-- hvplot
-- panel >=1
-- param
-- bokeh >=3
-- altair
-- pandas
+- python=3.11
+- notebook>=7.0.8
```

Please re-lock after that.

```suggestion
- notebook<7
```

---

Changed